### ISSUE TITLE
Auto-merging - issue?

### ISSUE THREAD

**Issue by flooie** [freelawproject/courtlistener#4720]
https://github.com/freelawproject/courtlistener/pull/4719

I was bumping juriscraper and set the PR to auto merge if all requirements were met.  The automerge completed and then a random unrelated recap test failed.  I'm not sure if this is a bug but I was under the impression that all tests needed to pass for auto merge to merge.

**Comment by flooie** [comment]
I guess that's as likely a scenario as there is. I was pretty sure I selected auto merge though.

**Comment by mlissner** [comment]
Yeah, who knows. I don't know how/why automerge got changed. I haven't played with the branch protection settings, have you?
